### DISCUSSION TITLE
Separate main site and tenant accounts setup - how to

### DISCUSSION THREAD

**Discussion by macumhail** [django-tenants/django-tenants#696] [Q&A]
I am working on a project with the following ideal structure. There will be a main site where account owners will have access to several community sections - forums for example. There will also be tenant accounts where the main site account owners will invite members to their tenant accounts and the tenant users will have access to only tenant sections. 

I added a wsgi_main_website.py file in the config directory with the following code from the docs (https://django-tenants.readthedocs.io/en/latest/install.html):
```
# wsgi_main_website.py
import os
os.environ.setdefault("DJANGO_SETTINGS_MODULE", "project.settings_public")

from django.core.wsgi import get_wsgi_application
application = get_wsgi_application()
```
I also added a settings_public.py file that only has settings for the main site (also per the documentation). 
I was able to get the main site to show after I added "SHOW_PUBLIC_IF_NO_TENANT_FOUND = True" to the main settings file. However, If I remove the forum apps for example,  from the SHARED_APPS I get an error even if the forum apps are installed in the settings_public.py file. 

If I leave the forum apps in shared apps then the tenant superusers have access to them. The tenant superusers even have admin access to the forum backend and could add or delete forums in the main site. 

Is is possible to set up django-tenants so that there is a main site with sections that are accessible to members but with those main site sections not available in tenant accounts. In other words if the main site had a section at www.mainsite.com/forums, there would be nothing at account1.mainsite.com/forums.

**Comment by Scratchydisk** [comment]
Hi, I have a similar problem.  Did you ever get an answer or work out a solution?
